docs/installation/cloud.md

```diff
@@ -0,0 +1,64 @@
+
+1. Create an account with the cloud provider, and read cloud provider documentation to understand their process for creating hosts.
+
+2. Decide which OS you want to run on the cloud host.
+
+3. Understand the Docker prerequisites and install process for the chosen OS. See [Install Docker Engine](index.md) for a list of supported systems and links to the install guides.
+
+4. Create a host with a Docker supported OS, and install Docker per the instructions for that OS.
+
+## Using Docker Machine to provision cloud hosts
+
+Docker Machine "drivers" are available for several popular cloud platforms, so you can use Docker Machine to provision Dockerized hosts on those platforms.
+
+To do so, you use the `docker-machine create` command with the `--driver` flag, along with provider-specific options for security credentials. Typically, these are the few required flags to create a system with the defaults. You can use optional flags to specify an operating system, machine and network configuration details, regions, and more.
+
+As an example, [Using Docker Machine with a cloud provider](https://docs.docker.com/machine/get-started-cloud/) walks you through the steps to provision a host on [Digital Ocean](https://www.digitalocean.com/) with the `digitalocean` driver.
+
+This is a great place to start.
+
+### Docker supported drivers
+
+__REVIEWERS NOTE:__ Right now the supported drivers topic lists several it sounds like we are going to drop support for. If so, we need to prune that list. It might be that Digital Ocean, AWS, and Azure, called out below, are the only ones we support going forward? When we surface those reference sections to task/concept topics (under Docker Machine guide?) we'll change these links to go to those new topics.
+
+Docker Machine drivers are available for several other cloud providers. For a full list, see [Supported Drivers](https://docs.docker.com/machine/drivers/).
+
+For details on how to use Docker Machine to provision hosts on these popular cloud platforms, see these reference topics.
+
+* [Amazon Web Services](https://docs.docker.com/machine/drivers/aws/)
+* [Digital Ocean](https://docs.docker.com/machine/drivers/digital-ocean/)
+* [Microsoft Azure](https://docs.docker.com/machine/drivers/azure/)
+
+
+### Docker Machine command and driver reference
+
+* `docker-machine` [create](https://docs.docker.com/machine/reference/create/) command
+* [Driver options and operating system defaults](https://docs.docker.com/machine/drivers/os-base/)
+
+### Third-party driver plugins
+A number of Docker Machine driver plugins for use with other cloud platforms are available from third-party contributors. These are use-at-your-own-risk plugins, not maintained by or formally associated with Docker.
+
+See [Available driver plugins](https://github.com/docker/machine/blob/master/docs/AVAILABLE_DRIVER_PLUGINS.md) in the docker/machine repo on GitHub.
+
+## Using Docker Machine on Mac and Windows cloud hosts
```

Comment:
I'd remove this for now.  I don't know that there are any cloud providers that server up Mac instances...and Windows VMs would need to install Toolbox so they could run machine and then they are running Vbox on a Vm in the Cloud to run Docker...which seems an edge case.  @nathanleclaire can speak to that --- I might be totally wrong.